### DISCUSSION TITLE
indoor_pressure_sensor: sensor.indoor_pressure

### DISCUSSION THREAD

**Discussion by lappat** [TheRealWaldo/ha-optimal-humidity#190] [Q&A]
i'm on version 2.0.5 and would like to, know if we don't have setup a indoor pressur sensor, it take the leevation per default of the home assistant?

- platform: optimal_humidity
  sensors:
    test_optimal_humidity:
      name: "Optimal Humidity"
      type: optimal_humidity
      indoor_temp_sensor: sensor.motion_sensor_1_air_temperature
      indoor_humidity_sensor: sensor.humiditymedian
      critical_temp_sensor: sensor.motion_sensor_3_air_temperature_3

![image](https://user-images.githubusercontent.com/61441238/214466287-02792c34-54da-4457-b0a4-e33385ddbf7d.png)

regards

**Comment by TheRealWaldo** [comment]
@lappat yes, it uses the `psycholib` library `GetStandardAtmPressure` with the elevation to assume the atmospheric pressure.  Unfortunately, that's a constant based on averages at that elevation, so it's only sometimes accurate.

Using a climate sensor, you can pull the air pressure from weather reports and get a more accurate level.

**Comment by lappat** [comment]
thanks for your reply,

i read somewhere  that in winter period between 30-40 max % is good, at the moment the new sensor is 43,3%, i have already setup a Mediam humidty in the house with 4 sensors.

![image](https://user-images.githubusercontent.com/61441238/214627260-cbd1a546-d810-41b6-99ff-b9e653a23ca6.png)

also what will be a good sensor for the climate sensor you talk about? at the moment i have a HVAC controller honneywel,

see picture, as you can see i d'ont have any other entity for, only the main one.

![image](https://user-images.githubusercontent.com/61441238/214626776-3fcc0602-66a3-4cfe-ac44-6158ac1ece98.png)

Thanks

**Reply by TheRealWaldo** [reply]
The readme explains how `ha-optimal-humidity` calculates the optimal humidity.  It's focussed more on human comfort, avoiding condensation, and avoiding mould, which a fixed Relative Humidity doesn't do a good job of.

I use https://www.home-assistant.io/integrations/environment_canada/ for external weather values; there are other options depending on where you are.

**Reply by lappat** [reply]
Hi,

just install yesterday and compare the value, at the moment I use the mediam humidty to start or stop my air exchanger with node red:

![image](https://user-images.githubusercontent.com/61441238/214637643-f47119d6-edc1-4aa3-93e2-e343bbf4b04c.png)

I have also setup in my HA environment Canada for external weather. Do I have to modify something to take-it in consideration?

I have only setup the 3 following sensors in sensors.yaml

indoor_temp_sensor: sensor.motion_sensor_1_air_temperature
indoor_humidity_sensor: sensor.humiditymedian
critical_temp_sensor: sensor.motion_sensor_3_air_temperature_3

I don't see any other parameter to setup?

Thanks for your help, I'm a newbie in programming and learning every day...

**Reply by TheRealWaldo** [reply]
I don't use node-red, but you'd need to take the pressure sensor they provide and drop it in your config as per the readme. While not perfect, the pressure indoors does not vary much from the outdoors. Using `indoor_pressure` is enough.

**Reply by lappat** [reply]
ok, i will test it and compare.

regards

**Comment by lappat** [comment]
also, i'm not able to find the blueprint, what is the adress?

![image](https://user-images.githubusercontent.com/61441238/214684365-ddbb5a31-5c08-4f9a-93cb-ebaf2e0beb89.png)

**Reply by TheRealWaldo** [reply]
I share my blueprints here: https://github.com/TheRealWaldo/ha-blueprints
